ui/tests/e2e/service_mesh_lifecycle.spec.js

```diff
@@ -0,0 +1,68 @@
+import { expect, test } from '@playwright/test';
+import { ENV } from './env';
+
+test.describe('Service Mesh Lifecycle Tests', () => {
+  const mesheryAdapters = [
+    { adapterName: 'Istio', adapterPort: '10000', deploy: false },
+    { adapterName: 'Consul', adapterPort: '10002', deploy: false },
+  ];
+
+  mesheryAdapters.forEach(({ adapterName, adapterPort, deploy }) => {
+    const ADAPTER_LOCATION = `localhost:${adapterPort}`;
+    test(`Configure Existing ${adapterName} adapter through Mesh Adapter URL from Management page`, async ({
+      page,
+    }) => {
+      // Visit Settings Page
+      await page.goto(`${ENV.MESHERY_SERVER_URL}/settings`);
+
+      // Navigate to 'Adapters' tab
+      await page.locator('[data-cy=tabServiceMeshes]').click({ force: true });
+
+      // Enter Mesh Adapter URL
+      await page.locator("[data-cy='mesh-adapter-url']").locator('input').fill('localhost:10000');
```

Comment:
dont use the data-cy or id locators , try to use the semantic locators from playwright

Reply:
Done